libs/nova-sms/src/lib/sms.service.ts

```diff
@@ -72,6 +73,9 @@ export class SmsService extends RESTDataSource {
   willSendRequest(request: RequestOptions) {
     this.memoizedResults.clear()
     request.headers.set('Content-Type', 'application/json')
+    request.agent = new Agent({
+      rejectUnauthorized: !this.options.acceptUnauthorized,
+    })
```

Comment:
Should be safe to reuse the agent between requests.

Reply:
Fixed.